### ISSUE TITLE
Inconsistent focus style

### ISSUE THREAD

**Issue by lkiesow** [elan-ev/tobira#663]
The border highlighting focused elements in Tobira is pretty inconsistent. Sometimes, you have rounded edges, sometimes you have not. Sometimes you have padding, sometimes you have not. Being consistent would be nice.

Example:

![Screenshot from 2023-01-17 13-46-10](https://user-images.githubusercontent.com/1008395/212903459-3e2730bd-bcba-421c-a310-e28670d57010.png)

**Comment by LukasKalbertodt** [comment]
Yeah, this pained me for a while now. I'm still unsure how to best fix it. For one, browsers have built-in focus styles that are used if not overwritten.

I would like a solution that just applies to (almost) everything. Otherwise, if we need to adjust it for every single UI element, we will always forget something and this issue will pop up again all the time. Anyone solving this issue should keep that in mind and think of how to define it as generally as possible.

There are two main problems that stopped me from writing a super easy general solution:

- Elements which have `overflow: hidden` can't have an outline outside of them. So they need an `outline-offset: -2px` or sth like that. But do we want that inset outline for all elements?! Maybe CSS has some crazy `*:[overflow="hidden"] { outline-offset: -2px }`? :D 
- Elements with rounded corners look bad when the outline is not rounded. That's usually not a problem except for elements like in the main navigation where the border radius is in the container, which also has `overflow:hidden` which only "cuts away" stuff from the first and last element. Currently we solve it like this:
  ```javascript
  "& > li:last-child > a": {
      borderRadius: `0 0 ${SIDE_BOX_BORDER_RADIUS}px ${SIDE_BOX_BORDER_RADIUS}px`,
  },
  "&:first-child > li:first-child > a": {
      borderRadius: `${SIDE_BOX_BORDER_RADIUS}px ${SIDE_BOX_BORDER_RADIUS}px 0 0`,
  },
  ```
  But this is obviously suboptimal as it's again something one has to do for these elements individually. Not sure if there is a good solution, to be honest.

One learning that I had over time is to use `overflow: hidden` as little as possible. But I don't think it's possible to avoid everywhere.

**Comment by LukasKalbertodt** [comment]
I think focus styles have improved a lot thanks to @owi92. As far as I can see, the only obvious things remaining are the focus ring around normal links (in text boxes and in the breadcrumbs). Those need to be rounded and maybe have a bit more padding (i.e. outline-offset).
